packages/experiments/src/components/Button/ButtonVariants.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { Button } from './Button';
+import { IButtonProps } from './Button.types';
+
+export interface IButtonVariantProps extends IButtonProps {
+  text?: string;
+}
+
+export type ButtonVariantsType = (props: IButtonVariantProps) => JSX.Element;
+
+export const DefaultButton: ButtonVariantsType = props => {
+  const { text, ...rest } = props;
+
+  return <Button content={text} {...rest} />;
+};
+
+export const PrimaryButton: ButtonVariantsType = props => {
```

Comment:
Maybe pretty minor now but I wonder if all variants are put in this file that it'll impact tree shaking's ability to remove unused ones?

Reply:
I can move them to their own files. Probably a better idea since I think some of them will require styling changes down the line.